src/gfn/gym/graph_building.py

```diff
@@ -245,18 +256,3 @@ def true_dist_pmf(self) -> torch.Tensor:
     def make_random_states_tensor(self, batch_shape: Tuple) -> GraphStates:
         """Generates random states tensor of shape (*batch_shape, feature_dim)."""
         return self.States.from_batch_shape(batch_shape)
-
```

Comment:
I forgot whether I left a comment on this here or not. So it's not needed after all ?

Reply:
No, it was just a mock default evaluator (nn to compute reward from states).
But it wouldn't be helpful for any user-case, it was just for easy testing GraphBuilding. 
I expect the user wants to always define the evaluator